desc/optimize/_scipy_wrappers.py

```diff
@@ -100,8 +100,7 @@ def hess_wrapped(xs):
         hess_allf.append(H)
         return H / (np.atleast_2d(x_scale).T * np.atleast_2d(x_scale))
 
-    def callback(xs):
-        x1 = xs / x_scale
```

Comment:
why was x_scale gotten rid of here?

Reply:
The callback is called from `grad_wrapped` after rescaling, so we don't need to rescale again